tests/Owin.Security.OpenIdConnect.Server.Tests/OAuth2AuthorizationClientCredentialsGrantTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Owin.Security.OpenIdConnect.Server.Tests
+{
```

Comment:
That would be great if you could use the K&R style to be consistent with the rest of the project :smile: